packages/components/src/illustrated-message/tests/chromatic/IllustratedMessage.chroma.jsx

```diff
@@ -0,0 +1,128 @@
+        .segment(segment)
+        .build();
+}
+
+stories()
+    .add("default", () =>
+        <IllustratedMessage>
+            <Image src={Nasa} alt="Nasa" width="150px" />
+            <Heading>Can't find "Saturn"</Heading>
+            <Content>Try searching for something else.</Content>
+        </IllustratedMessage>
+    )
+    .add("horizontal", () =>
+        <Stack>
+            <IllustratedMessage orientation="horizontal" width="700px" height="200px">
+                <Image src={Nasa} alt="Nasa" />
+                <Heading>Can't find "Saturn"</Heading>
+                <Content>Try searching for something else.</Content>
+            </IllustratedMessage>
+            <Box width="700px" height="200px">
+                <IllustratedMessage orientation="horizontal">
+                    <Image src={Nasa} alt="Nasa" />
+                    <Heading>Can't find "Saturn"</Heading>
+                    <Content>Try searching for something else.</Content>
+                </IllustratedMessage>
+            </Box>
+        </Stack>
+    )
+    .add("vertical", () =>
+        <Inline>
+            <IllustratedMessage orientation="vertical" width="200px" height="500px">
+                <Image src={Nasa} alt="Nasa" />
+                <Heading>Can't find "Saturn"</Heading>
+                <Content>Try searching for something else.</Content>
+            </IllustratedMessage>
+            <Box width="200px" height="500px">
+                <IllustratedMessage orientation="vertical">
+                    <Image src={Nasa} alt="Nasa" />
+                    <Heading>Can't find "Saturn"</Heading>
+                    <Content>Try searching for something else.</Content>
+                </IllustratedMessage>
+            </Box>
+        </Inline>
+    )
+    .add("very long title", () =>
+        <IllustratedMessage width="700px" height="200px">
+            <Image src={Nasa} alt="Nasa" />
+            <Heading>Can't find "Saturn" or "Mars" or another continent.</Heading>
+            <Content>Try searching for something else.</Content>
+        </IllustratedMessage>
+    )
+    .add("very long content", () =>
+        <IllustratedMessage width="700px" height="200px" orientation="horizontal">
+            <Image src={Nasa} alt="Nasa" />
+            <Heading>Can't find "Saturn"</Heading>
+            <Content>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc suscipit metus neque, non pharetra enim tincidunt dictum. Fusce in ultricies turpis, vitae finibus nunc. Quisque laoreet sit amet eros eget volutpat. Pellentesque non nulla dui. Sed nec felis quam. Vestibulum velit magna, fringilla ut neque cursus, porta rhoncus nulla. Suspendisse auctor sollicitudin tortor, quis viverra tellus egestas sed. Pellentesque ut dignissim nisi. Duis sit amet ex bibendum, pharetra purus eget, varius massa. In pulvinar dui quis dignissim commodo. Nulla facilisi..</Content>
+        </IllustratedMessage>
+    )
+    .add("no title", () =>
```

Comment:
I don't think we should allow this use case.

Reply:
The components allows it, I can see some uses cases where a title would be too much and some italic explanation would be enough.

Reply:
I would validate with the designers. I personally would see those cases as out of brand.